plugins/modules/aci_l3out_static_routes_nexthop.py

```diff
@@ -48,6 +48,10 @@
     description:
     - The nexthop for the prefix
     type: str
+  preference:
+    description:
+    - The administrative preference value for the nexthop
+    type: int
```

Comment:
Could you document the default value apic sets when not provided and the allowed integer range, think it is 1-255 if not mistaken. Could you validate this and adjust the docs suggestion below.

```suggestion
  preference:
    description:
    - The administrative preference value for the nexthop.
    - The APIC defaults to 0 when unset during creation.
    - The value must be between 1 and 255.
    type: int
```

Reply:
Sure 0-255 is fine, if you can adjust that it is good.

Reply:
Fixed. Please confirm. Thank you!